<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>X2Dome</class>
 <widget class="QDialog" name="X2Dome">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>712</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>712</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>NexDome controller settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="X2Form" name="X2Form">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1024</width>
       <height>1024</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="SerialPortName" stdset="0">
      <string/>
     </property>
     <widget class="QGroupBox" name="MaxDomeIIParams">
      <property name="geometry">
       <rect>
        <x>16</x>
        <y>80</y>
        <width>296</width>
        <height>176</height>
       </rect>
      </property>
      <property name="title">
       <string>Dome params</string>
      </property>
      <widget class="QCheckBox" name="homeOnUnpark">
       <property name="geometry">
        <rect>
         <x>13</x>
         <y>144</y>
         <width>140</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Home after unpark</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="hasShutterCtrl">
       <property name="geometry">
        <rect>
         <x>13</x>
         <y>96</y>
         <width>183</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Shutter controller present</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="homeOnPark">
       <property name="geometry">
        <rect>
         <x>13</x>
         <y>120</y>
         <width>136</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Home before park</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>32</y>
         <width>160</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Home Position (Deg.) :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="homePosition">
       <property name="geometry">
        <rect>
         <x>184</x>
         <y>32</y>
         <width>88</width>
         <height>24</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="maximum">
        <double>359.990000000000009</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>32</x>
         <y>64</y>
         <width>144</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Park Postition (Deg.) :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="parkPosition">
       <property name="geometry">
        <rect>
         <x>184</x>
         <y>64</y>
         <width>88</width>
         <height>24</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="maximum">
        <double>359.990000000000009</double>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>336</x>
        <y>80</y>
        <width>336</width>
        <height>320</height>
       </rect>
      </property>
      <property name="title">
       <string>Controller settings</string>
      </property>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>56</x>
         <y>64</y>
         <width>160</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Rotation speed (step/s) :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QSpinBox" name="rotationSpeed">
       <property name="geometry">
        <rect>
         <x>224</x>
         <y>64</y>
         <width>104</width>
         <height>24</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="rotationAcceletation">
       <property name="geometry">
        <rect>
         <x>224</x>
         <y>96</y>
         <width>104</width>
         <height>24</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>96</y>
         <width>200</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Rotation acceleration (ms) :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>8</x>
         <y>32</y>
         <width>208</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Rotator steps per revolution :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QSpinBox" name="ticksPerRev">
       <property name="geometry">
        <rect>
         <x>224</x>
         <y>32</y>
         <width>104</width>
         <height>24</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="shutterAcceleration">
       <property name="geometry">
        <rect>
         <x>224</x>
         <y>240</y>
         <width>104</width>
         <height>24</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>240</y>
         <width>200</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Shutter acceleration (ms) :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>56</x>
         <y>208</y>
         <width>160</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Shutter speed (step/s) :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QSpinBox" name="shutterSpeed">
       <property name="geometry">
        <rect>
         <x>224</x>
         <y>208</y>
         <width>104</width>
         <height>24</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="shutterTicks">
       <property name="geometry">
        <rect>
         <x>224</x>
         <y>176</y>
         <width>104</width>
         <height>24</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>8</x>
         <y>176</y>
         <width>208</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Shutter steps per open/close :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>32</x>
         <y>280</y>
         <width>281</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Reset to factory default</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>8</x>
         <y>128</y>
         <width>208</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Rotator dead zone (steps) :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QSpinBox" name="rotDeadZone">
       <property name="geometry">
        <rect>
         <x>224</x>
         <y>128</y>
         <width>104</width>
         <height>24</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="maximum">
        <number>2000</number>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_logo">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>0</y>
        <width>128</width>
        <height>44</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>NexDome.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="X2_PhotoFileName" stdset="0">
       <string>NexDome.png</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>48</y>
        <width>168</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>© RTI-Zone.org, 2019</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonCancel">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>416</y>
        <width>81</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonOK">
      <property name="geometry">
       <rect>
        <x>576</x>
        <y>416</y>
        <width>81</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>16</x>
        <y>256</y>
        <width>296</width>
        <height>160</height>
       </rect>
      </property>
      <property name="title">
       <string>Status</string>
      </property>
      <widget class="QLabel" name="rainStatus">
       <property name="geometry">
        <rect>
         <x>176</x>
         <y>72</y>
         <width>96</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
      <widget class="QLabel" name="shutterBatteryLevel">
       <property name="geometry">
        <rect>
         <x>176</x>
         <y>48</y>
         <width>56</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>12.25 V</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>72</y>
         <width>88</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Rain sensor :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>24</x>
         <y>48</y>
         <width>144</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Shutter battery level :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>24</x>
         <y>24</y>
         <width>144</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Current step position :</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="currentStepPos">
       <property name="geometry">
        <rect>
         <x>176</x>
         <y>24</y>
         <width>96</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>123456789</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>96</y>
         <width>152</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Log rain sensor state</string>
       </property>
      </widget>
      <widget class="QLabel" name="filePath">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>120</y>
         <width>264</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>/Users/foo/NDV3_Rain.txt</string>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>X2Form</class>
   <extends>QFrame</extends>
   <header>x2form.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>8</number>
  </property>
  <property name="gridDeltaY">
   <number>8</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
